openstack/compute/v2/extensions/quotasets/testing/fixtures.go

```diff
@@ -2,13 +2,12 @@ package testing
 
 import (
 	"fmt"
-	"net/http"
-	"testing"
-
+	"github.com/Comcast/gophercloud/openstack/compute/v2/extensions/quotasets"
 	"github.com/gophercloud/gophercloud"
-	"github.com/gophercloud/gophercloud/openstack/compute/v2/extensions/quotasets"
 	th "github.com/gophercloud/gophercloud/testhelper"
 	"github.com/gophercloud/gophercloud/testhelper/client"
+	"net/http"
+	"testing"
```

Comment:
Looks like this needs cleaned up: `Comcast` needs to be replaced with `gophercloud` (which is the cause of Travis failing). As well, the imports need to be re-arranged to the original ordering: std packages first, blank line, then github packages.